server/hedley/modules/custom/hedley_admin/scripts/assign-nutrition-encounter-type.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/**
+ * @file
+ * Assigns Nutrition encounter type to all encounters that got measurements.
+ *
+ * If measurement created by nurse - encounter type
+ * set to nurse. If by CHW, set to chw.
+ *
+ * Drush scr
+ * profiles/hedley/modules/custom/hedley_admin/scripts/assign-nutrition-encounter-type.php.
+ */
+
+if (!drupal_is_cli()) {
+  // Prevent execution from browser.
+  return;
+}
+
+// Get the number of nodes to be processed.
+$batch = drush_get_option('batch', 500);
```

Comment:
Is this batch site safe enough? Tested with live DB?

Reply:
yep